WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2812,6 +2768,18 @@ public void done() {
                         savePostAsync(null);
                     }
                     break;
+                case RequestCodes.REVISION_DETAILS:
+                    if (data.hasExtra(KEY_REVISION)) {
+                        mViewPager.setCurrentItem(PAGE_CONTENT);
+
+                        mRevision = data.getParcelableExtra(KEY_REVISION);
+                         new Handler().postDelayed(new Runnable() {
```

Comment:
```suggestion
                        new Handler().postDelayed(new Runnable() {
```

Can we remove the extra space for code style consistency?